packages/cardano-services/README.md

```diff
@@ -64,28 +72,54 @@ _The following examples require the [install and build] steps to be completed._
 - HTTP API exposed using a custom API URL
 - Prometheus metrics exporter enabled at http://localhost:6000/metrics
 
+**`start-server` using CLI options:**
 ``` console
 ./dist/cjs/cli.js \
   start-server \
     --api-url http://localhost:6000 \
-    --cardano-node-config-path ./config/cardano-node/config.json \
     --enable-metrics true \
-    --ogmios-srv-service-name  some-domain-for-ogmios \
-    --postgres-db someDbName \
-    --postgres-password somePassword \
+    --cardano-node-config-path ./config/network/testnet/cardano-node/config.json \
     --postgres-srv-service-name \
+    --postgres-db someDbName \
     --postgres-user somePgUser \
+    --postgres-password somePassword \
+    --ogmios-srv-service-name  some-domain-for-ogmios \
     --rabbitmq-srv-service-name some-domain-for-rabbitmq \
     --use-queue \
-    asset,chain-history,stake-pool,tx-submit,network-info,utxo
+    asset,chain-history,stake-pool,tx-submit,network-info,utxo,rewards
```

Comment:
```suggestion
    asset chain-history stake-pool tx-submit network-info utxo rewards
```

Reply:
That wrong CLI usage in readme was introduced before my change so I assume it was due to misleading similarity between passing service names through `cli` and `run` .ts (run.ts uses commas or space separation).

Done in 8a21af9a2376eb03073ef97c909f5514dff135c4 @iadmytro